crypto/src/binprot/mod.rs

```diff
@@ -1,22 +1,23 @@
 // Copyright 2020 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::de::Deserializer;
+pub use bin_prot::error::Error;
+use bin_prot::Deserializer;
 use serde::{Deserialize, Serialize};
 
-pub trait BinProtDeser {
+pub trait BinProtEncodable {
```

Comment:
Ok I understand why this needs to be BinProt specific now, since then you can know exactly how much buffer to allocate